docs/source/install.rst

```diff
@@ -0,0 +1,225 @@
+
+- Close the PowerShell and then re-open another PowerShell (Admin) by repeating the first two steps.
+- Install Python and PyCharm
+
+.. code-block:: bash
+
+  choco install cmake git python pycharm-community -y
+
+Enabling the USB device (only on Linux)
+#######################################
+
+Since the DepthAI is a USB device, in order to communicate with it on the systems that use :code:`udev` tool, you
+need to add the udev rules in order to make the device accessible.
+
+The following command will add a new udev rule to your system
+
+.. code-block:: bash
+
+  echo 'SUBSYSTEM=="usb", ATTRS{idVendor}=="03e7", MODE="0666"' | sudo tee /etc/udev/rules.d/80-movidius.rules
+  sudo udevadm control --reload-rules && sudo udevadm trigger
+
+Install using pip
+#################
+
+Our packages are available to download using our `Artifactory server <https://artifacts.luxonis.com/artifactory/luxonis-python-snapshot-local/>`__.
+These are built for every commit in the `depthai-python <https://github.com/luxonis/depthai-python/tree/gen2_develop>`__ repository and are suffixed
+with a commit SHA.
+
+Note that we recommend installing the dependencies in a virtual environment, so that they don't interfere with other Python
+tools/environments on your system.
+
+- For development machines like Mac/Windows/Ubuntu/etc., we recommend the `PyCharm <https://www.jetbrains.com/pycharm/>`__ IDE, as it automatically makes/manages virtual environments for you, along with a bunch of other benefits.  Alternatively, :code:`conda`, :code:`pipenv`, or :code:`virtualenv` could be used directly (and/or with your preferred IDE).
+- For installations on resource-constrained systems, such as the Raspberry Pi or other small Linux systems, we recommend :code:`conda`, :code:`pipenv`, or :code:`virtualenv`.  To set up a virtual environment with :code:`virtualenv`, run
+
+  .. code-block:: bash
+
+    virtualenv venv
+    source venv/bin/activate
+
+Using a virtual environment (or system-wide, if you prefer), run the following to install the DepthAI using pip:
+
+#. Pick the preferred commit from `commit list <https://github.com/luxonis/depthai-python/commits/gen2_develop>`__
+
+#. Upgrade pip
+
+  .. code-block:: bash
+
+    python3 -m pip install -U pip
+
+#. Install DepthAI Gen2 API, adding commit SHA at the end of a package name, in form of :code:`depthai==0.0.2.1+<sha>`
+
+  .. code-block:: bash
+
+    python3 -m pip install --extra-index-url https://artifacts.luxonis.com/artifactory/luxonis-python-snapshot-local/ depthai==0.0.2.1+6ec3f3181b4e46fa6a9f9b20a5b4a3dac5e876b4
+
+For other installation options, see :ref:`other installation options <Other installation methods>`.
+
+Test installation
+#################
+
+We have `a set of examples <https://github.com/luxonis/depthai-python/tree/gen2_develop/examples>`__ that should help you verify if your setup was correct.
```

Comment:
We can create an archive of examples which include the necessary data alongside it, and deploy that to artifactory.
Just an idea for now

Reply:
Then it's better to separate the examples from API - which will be ultimately the `depthai` repo